frontend/src/metabase/lib/form.ts

```diff
@@ -0,0 +1,19 @@
+import _ from "lodash";
+import { BaseFieldDefinition } from "metabase-types/forms";
+
+export function createErrorMessageMarkdown(
+  message?: string,
+  formFields?: BaseFieldDefinition[],
+): string | undefined {
+  if (!message) {
+    return undefined;
+  }
+
+  const fieldTitles = (formFields || [])
+    .map(formField => formField.title)
+    .filter(_.identity) as string[];
+
+  return fieldTitles.reduce((transformedMessage, fieldTitle) => {
+    return transformedMessage.replaceAll(fieldTitle, `**${fieldTitle}**`);
```

Comment:
Initially, I was doing
```tsx
return transformedMessage.replaceAll(fieldTitle, `**${t`${fieldTitle}`}**`);
```
but the [CI failed](https://github.com/metabase/metabase/runs/6523316641?check_suite_focus=true). 

I also observed that BE set localized database field names inside a JSON. So, I assumed BE also needs to localize the error message, not FE? Since it's not that there's localization for field `Host` that translates to `Hôte` but rather `Hôte` is in the `display-name` for database engines.

Not sure if I missed anything. Any thoughts @metamben @ranquild 
![image](https://user-images.githubusercontent.com/1937582/169516063-de9f16e5-b606-4c81-aadd-ca54b00096c2.png)

Reply:
@WiNloSt New localization strings aren't yet translated. They will be before the release

Reply:
Until then new messages would be always in English